R/extract_features.R

```diff
@@ -97,15 +103,31 @@ InputExample_EF <- function(unique_id,
 #' @return An \code{input_fn} closure to be passed to TPUEstimator.
 #' @keywords internal
 input_fn_builder_EF <- function(features,
-                             seq_length) {
-  all_unique_ids <- purrr::map(features,
-                              function(f) { as.integer(f$unique_id) })
-  all_input_ids <- purrr::map(features,
-                               function(f) { as.integer(f$input_ids) })
-  all_input_mask <- purrr::map(features,
-                                function(f) { as.integer(f$input_mask) })
-  all_input_type_ids <- purrr::map(features,
-                              function(f) { as.integer(f$input_type_ids) })
+                                seq_length) {
+  all_unique_ids <- purrr::map(
```

Comment:
This is one of the places that styler makes me cry. A code chunk that should *ideally* be four lines long is now more than twenty.

Reply:
When I'm reviewing this sort of code, one of the first things I do is to break it into multiple lines so I don't get lost 🙃